### DISCUSSION TITLE
NXP  i.MX RT595 Evaluation Kit driver support

### DISCUSSION THREAD

**Discussion by jdwang001** [zephyrproject-rtos/zephyr#49788] [Q&A] [answered]
Our existing program is implemented based on nordic sdk (zephyr rtos).
At this stage, we have  i.MX RT595 Evaluation Kit development board, and we want to quickly port the program to this board.
We are more concerned about the follow-up driver support of rt595 in zephyr.
Mainly the mipi screen, and the GPU driver part.
So, is there a specific time for related driver development?

**Comment by DerekSnell** [comment, marked as answer]
Hi @jdwang001 ,

NXP does have plans to add driver support for both the MIPI DSI display interface, and the GPU.  However, these tasks are not yet scheduled, so we do not have specific timing for those features to share yet.

In the meantime, the RT1170 has a similar MIPI DSI feature that is already supported in Zephyr.  You can refer to this [PR ](https://github.com/zephyrproject-rtos/zephyr/pull/36875) as a reference for what was added to enable this support.

For the GPU, my understanding is Zephyr does not have a framework yet for GPU support.  NXP has multiple MCUs supported in Zephyr today with GPUs or some graphical acceleration.  So we are quite interested in working with the Zephyr Community to create a common framework that can be used.  And then NXP plans to add this support for the RT500 family.

Best regards

**Reply by jdwang001** [reply]
THX @DerekSnell
